packages/fether-react/src/Accounts/CreateAccount/AccountName/AccountName.js

```diff
@@ -33,6 +33,18 @@ class AccountName extends Component {
     return isImport ? this.renderCardWhenImported() : this.renderCardWhenNew();
   }
 
+  handleKeyPress = e => {
```

Comment:
Surely there's some better way to handle this? I.e. within final-form.

If too complicated, then leave it like this.

Reply:
Final form doesn't have any keypress handlers AFAICT. I agree this is nasty but not sure that there is a way around it.